riot/account/account.go

```diff
@@ -0,0 +1,53 @@
+package account
+
+import (
+	"fmt"
+	"github.com/KnutZuidema/golio/api"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/KnutZuidema/golio/internal"
+)
+
+// AccountClient provides methods for the account endpoints of the League of Legends API.
+type AccountClient struct { //nolint:golint
```

Comment:
@KnutZuidema in the other PR you mentioned this should be renamed to `Client`. However, `account.Client` already exists in [account/client.go](https://github.com/KnutZuidema/golio/pull/60/files#diff-e7bdc4af264de5c189e54d0eaf81796f1bf0980087ade9ac9afb9fb4fe93d484R6-R8) so that this package conforms to the same structure in the `lol` package.

I can change this if you want, but if another `account`-family endpoint(s) is added by riot at some point it may be useful to keep this structure? Up to you

Reply:
Fair point. I would assume that the endpoint hierarchy for accounts will stay flat, so let's just implement all the actions directly on the `Client` type. If another set of endpoints is added later, we'll have to refactor.

Reply:
[1bce0c4](https://github.com/KnutZuidema/golio/pull/60/commits/1bce0c4b7da4f918608bf228137d61631cf35cea)

Reply:
[1bce0c4](https://github.com/KnutZuidema/golio/pull/60/commits/1bce0c4b7da4f918608bf228137d61631cf35cea)